packages/frontend/src/setupTests.js

```diff
@@ -14,3 +22,17 @@ afterEach(() => mswServer.resetHandlers());
 afterAll(() => {
   mswServer.close();
 });
+
+export const Providers = ({ children }) => (
+  <BrowserRouter>
+    <FormProvider>
+      <PhotosProvider>
+        <GameProvider>{children} </GameProvider>
+      </PhotosProvider>
+    </FormProvider>
+  </BrowserRouter>
+);
```

Comment:
I feel like this should be exported from `helpers/tests.jsx` because that's where the `setupTests` function is exported from and it could also be used there where `render` is called.